Cargo.toml

```diff
@@ -15,6 +16,7 @@ members = [
     "primitives/sidechain/primitives",
     "primitives/storage",
     "primitives/test-utils",
+    "primitives/tls_websocket_server",
```

Comment:
new crate for the TLS websocket server (uses SGX ported Tungstenite library under the hood)